projects/protobuf.cmake

```diff
@@ -1,8 +1,8 @@
 # protobuf
 xpProOption(protobuf DBG)
-set(VER 3.0.0-beta-1)
+set(VER 3.13.0)
 set(REPO github.com/protocolbuffers/protobuf)
-set(FORK github.com/smanders/protobuf)
+set(FORK github.com/cameronfrandsen/protobuf)
```

Comment:
This is something that will definitely have to change when the other pull is merged. I will revert it back to smanders

Reply:
technically you can make the patch without changing this line... just make a remote in the cloned repo and make the patch -- but this is fine, I can deal with this